control/init.sh

```diff
@@ -0,0 +1,61 @@
+#! /bin/bash
```

Comment:
This file seems to copy a lot of stuff from ansible-dotfiles. Is the idea that this will replace that?